js/adapt-contrib-mcq.js

```diff
@@ -166,20 +166,16 @@ define(function(require) {
                     // If an item shouldn't be selected and is selected
                     // Adjust incorrect answers
                     numberOfIncorrectAnswers ++;
-
                 }
 
             }, this);
 
             this.model.set('_numberOfCorrectAnswers', numberOfCorrectAnswers);
+            this.model.set('_numberOfRequiredAnswers', numberOfRequiredAnswers);
 
             // Check if correct answers matches correct items and there are no incorrect selections
-            if (numberOfCorrectAnswers === numberOfCorrectItems && numberOfIncorrectAnswers === 0) {
-                return true;
-            } else {
-                return false;
-            }
-            
+            var answeredCorrectly = numberOfCorrectAnswers === numberOfRequiredAnswers && numberOfIncorrectAnswers === 0;
```

Comment:
Could we have this written as ``(numberOfCorrectAnswers === numberOfRequiredAnswers) && (numberOfIncorrectAnswers === 0);`` - makes it easier to read as your evaluating